src/main/scala/com/horizon/exchangeapi/OrgsRoutes.scala

```diff
@@ -1231,7 +1233,7 @@ trait OrgsRoutes extends JacksonSupport with AuthenticationSupport {
         )),
       new responses.ApiResponse(responseCode = "400", description = "bad input"),
       new responses.ApiResponse(responseCode = "401", description = "invalid credentials"),
-      new responses.ApiResponse(responseCode = "403", description = "access denied")))
```

Comment:
This needs to be added back in. The permission `READ_MY_ORG` can be removed from the running Exchange config, throwing a 403 for any user archetype that calls this route.

Reply:
ok, makes sense